src/PoolManager.sol

```diff
@@ -323,16 +323,13 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
     function _accountDelta(Currency currency, int128 delta, address target) internal {
         if (delta == 0) return;
 
-        int256 current = currency.getDelta(target);
-        int256 next = current + delta;
+        (int256 previous, int256 updated) = currency.applyDelta(target, delta);
```

Comment:
nit: use `next` variable instead of updated to be consistent with return names

Reply:
One function's var name is not necessarily another function's var name; the variables are in different contexts. Are you sure you prefer the name next for this function? If they must be named the same, I would rather call the inner function's variable updated.

Reply:
I know that, its just a neater practice and the first variable `previous` is called the same thing in both contexts so why would we not make the second one also consistent? 

I actually prefer the variable name `next`. It is more synonymous with `previous`.

Reply:
2ffe0195